src/main/java/racingcar/controller/RacingController.java

```diff
@@ -0,0 +1,68 @@
+package racingcar.controller;
+
+import racingcar.domain.Car;
+import racingcar.domain.Cars;
+import racingcar.dto.CarDto;
+import racingcar.service.ParseCarNames;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+import java.util.List;
+
+public class RacingController {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final ParseCarNames parseCarNames;
+
+    public RacingController(InputView inputView, OutputView outputView, ParseCarNames parseCarNames) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.parseCarNames = parseCarNames;
+    }
+
+    public void run() {
+        Cars cars = RacingStart();
+
+        moveCar(cars);
+
+        List<String> winner = getWinner(cars);
+
+        printWinner(winner);
+
+        inputView.closeConsole();
+    }
+
+    public void printWinner(List<String> winner){
+        outputView.printWinner(winner);
+    }
+
+    public List<String> getWinner(Cars cars) {
+        List<CarDto> winner = cars.findWinner();
+        return winner.stream().
+                map(CarDto::getName)
+                .toList();
```

Comment:
같은 클래스에서만 사용되는 메서드는 private로 작성해보시면 어떨까요?? ( 캡슐화)

```suggestion

    private List<String> getWinner(Cars cars) {
        List<CarDto> winner = cars.findWinner();
        return winner.stream().
                map(CarDto::getName)
                .toList();
```